spec/amp-actions-and-events.md

```diff
@@ -362,6 +362,22 @@ event.response</pre></td>
   </tr>
 </table>
 
+### amp-selector
+<table>
+  <tr>
+    <th>Action</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td><code>selectUp(delta=INTEGER)</code></td>
+    <td>Changes the selected element to the currentPos - `delta`. The default `delta` is set to 1.</td>
```

Comment:
currentPos isn't a variable so it seems kind of out of context. Consider changing the wording to  something like "Moves the selection up by `delta`"